diagnostic.md

```diff
@@ -5,74 +5,88 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+It defines/creates the framework of the app
 ```
```

Comment:
`One purpose of a backend is to allow users on the same web app to interact with each other.
Without a backend, users would not be able to do this, limiting what our application is capable of. Another
purpose is to securely store data that can persist over time.`